pypiserver/Tiltfile

```diff
@@ -0,0 +1,66 @@
+EXT_PATH = os.getcwd()
+PYPI_PORT_VAR_NAME = 'PYPISERVER_PORT' # docker compose purposes
+DEFAULT_PYPI_PORT_VALUE = '8222'
+DEFAULT_CACHE_DIR = '.tilt-cache'
+DEFAULT_PYPI_CONTAINER_NAME = 'tilt-pypiserver'
+PYPI_CONTAINER_VAR_NAME = 'PYPI_CONTAINER_NAME' # docker compose purpose
+PYPISERVER_DATA_CACHE_DIR_VAR_NAME = 'PYPISERVER_PACKAGES_DATA_CACHE' # docker compose purposes
+PYPISERVER_DATA_CACHE_DIR = 'pypiserver'
+
+PYTHON_CMD = 'python3'
+
+def _root_tiltfile_dir():
```

Comment:
this is slightly incorrect, can you copy the implementation from `git_resource`?

Reply:
But the one from `git_resource` does not work ([test](https://app.circleci.com/pipelines/github/tilt-dev/tilt-extensions/1529/workflows/d4b7b1bc-5f26-49d8-8d00-1defdfb37023/jobs/3651)), it passes the tests only because there is [this](https://github.com/korbajan/tilt-extensions/blob/pypiserver/git_resource/test/Tiltfile#L1) setting in the test/Tiltfile :)